src/chunk_index.c

```diff
@@ -728,13 +727,13 @@ chunk_index_delete_by_hypertable_id(int32 hypertable_id, bool drop_index)
 								   scankey, 1, chunk_index_tuple_delete, NULL, &data);
 }
 
-static bool
+static ScanTupleResult
 chunk_index_tuple_found(TupleInfo *ti, void *const data)
 {
 	ChunkIndexMapping *const cim = data;
 
 	chunk_index_mapping_from_tuple(ti, cim);
-	return false;
+	return SCAN_CONTINUE;
```

Comment:
Another semantic change, this used to access only one tuple, and will now access all of them.

Reply:
Yeah, this was just a bad change and has been fixed.